module/VuFind/src/VuFind/View/Helper/Root/SearchMemory.php

```diff
@@ -48,14 +50,25 @@ class SearchMemory extends AbstractHelper
      */
     protected $memory;
 
+    /**
+     * Search table
+     *
+     * @var Search
+     */
+    protected $searchTable;
+
     /**
      * Constructor
      *
      * @param Memory $memory Search memory
      */
-    public function __construct(Memory $memory)
+    public function __construct(Memory $memory, Search $search, string $sessionId,
+        \VuFind\Search\Results\PluginManager $resultsManager)
     {
         $this->memory = $memory;
+        $this->searchTable = $search;
+        $this->sessionId = $sessionId;
```

Comment:
Is this work in progress? You seem to have added dependencies that aren't used yet.

Reply:
That was left over after moving the actual functionality to \VuFind\Search\Memory. Now cleaned up!